portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,376 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+@import url('https://fonts.googleapis.com/css2?family=Comfortaa:wght@300;400&display=swap');
```

Comment:
It is recommended to omit the quotation marks: https://google.github.io/styleguide/htmlcssguide.html#CSS_Quotation_Marks

Reply:
okay, will change.